help/develop/javascript-development/datagrok-configuration.md

```diff
@@ -0,0 +1,39 @@
+# Datagrok configuration
+
+After you install `datagrok-tools` and run the first command (probably, to [generate a package]), you need to configure
+your environment.
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Developer Keys](#developer-keys)
+
+## Overview
+
+The Grok CLI creates a configuration file under `%USERNAME%/.grok/config.yaml` to store the configurations for your
+server, developer keys, and other metadata that's used by Datagrok.
+
+You can configure this file from the terminal by running `grok config` and then entering the needed configurations. 
+Alternatively, you can change this file manually.
+
+## Developer Keys
+
+To develop your packages or applications on Datagrok, you need to obtain the developer keys.
+
+## How to obtain the public developer key
+
+To obtain the public developer key, open [your user profile on the public server] and then click **Developer key**.
+Datagrok administrators can manage existing keys and grant or revoke privileges.
```

Comment:
This statement needs to be clarified a bit (how does it work, are there any implications, what does the workflow look like), don't you think so?